test/unit/math/rev/fun/binomial_coefficient_log_test.cpp

```diff
@@ -0,0 +1,346 @@
+#include <stan/math/prim.hpp>
+#include <stan/math/rev.hpp>
+#include <test/unit/math/expect_near_rel.hpp>
+#include <gtest/gtest.h>
+#include <limits>
+#include <algorithm>
+#include <vector>
+#include <string>
+#include <cmath>
+
+TEST(MathFunctions, binomial_coefficient_log_identities) {
```

Comment:
What's the identity being tested?

Reply:
Added as a comment. It is `(n choose k) == (n - 1 choose k - 1) * n/k`